aicsimageio/vendor/omexml.py

```diff
@@ -35,16 +34,16 @@ def xsd_now():
 #
 # The namespaces
 #
-NS_BINARY_FILE = "http://www.openmicroscopy.org/Schemas/BinaryFile/2013-06"
+NS_BINARY_FILE = "http://www.openmicroscopy.org/Schemas/BinaryFile/2016-06"
```

Comment:
I think this is great but should this perhaps be in its own fork and then that fork can be included until the changes are pushed back to Broad?

Reply:
Dan and I have talked about this a bit. Our vendor file has already diverged from the Broad origin a while back. I think the plan in my head is to fork not the Broad one but the `OME/ome_model` repo or something prior or during the 4.0 writers release.